### DISCUSSION TITLE
Dịch `agent`

### DISCUSSION THREAD

**Discussion by rootonchair** [mlbvn/handson-ml2-vn#11] [Terminology] [answered]
Lúc trước `agent` trong RL mình hay dịch là `tác nhân`. Nhưng em biết từ này thường được dịch là  `tác tử`. Mọi người thấy mình nên update lại không?

https://huuvinhfit.files.wordpress.com/2015/01/chuong-2-tac-tu.pdf

cc: @ngcthuong @rootonchair @nguyenhoa93 @quangnhat185 @minhduc0711 @davidnvq @cuongvng @duythanhvn

**Comment by lkhphuc** [comment]
Vote `Tác tử`

**Comment by thanhcsf** [comment, marked as answer]
Từ này tạm giữ là `tác nhân` mọi người nha.

**Reply by minhduc0711** [reply]
@duythanhvn em thấy mọi người vote `tác tử` mà anh. Trước em có làm về `agent-based model`, cũng được dịch là `mô hình đa tác tử`.

**Reply by thanhcsf** [reply]
Chưa hẳn là vote em ạ https://github.com/mlbvn/handson-ml2-book/pull/32#discussion_r566783098
